hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -193,6 +196,32 @@ public TransactionContext startTransaction(
     return handleStartTransactionRequests(raftClientRequest, omRequest);
   }
 
+  public TransactionContext preAppendTransaction(TransactionContext trx)
```

Comment:
Can you add @Override here, so we could directly know this is an override method.

Reply:
Done